ci-operator/step-registry/hypershift/kubevirt/install/hypershift-kubevirt-install-commands.sh

```diff
@@ -6,18 +6,31 @@ function ocp_version() {
     oc get clusterversion version -o jsonpath='{.status.desired.version}' | awk -F "." '{print $1"."$2}'
 }
 
-CNV_PRERELEASE_VERSION=${CNV_PRERELEASE_VERSION:-$(ocp_version)}
-CNV_PRERELEASE_CATALOG_IMAGE=${CNV_PRERELEASE_CATALOG_IMAGE:-quay.io/openshift-cnv/nightly-catalog:${CNV_PRERELEASE_VERSION}}
+# Get yq tool
+YQ="/tmp/yq"
+curl -L -o ${YQ} https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
+chmod +x ${YQ}
+
+# Dynamically get CNV catalog image that was provided to the job via gangway API
+CNV_PRERELEASE_CATALOG_IMAGE=$(curl -s https://prow.ci.openshift.org/prowjob?prowjob="${PROW_JOB_ID}" |\
+  ${YQ} e '.spec.pod_spec.containers[0].env[] | select(.name == "CNV_PRERELEASE_CATALOG_IMAGE") | .value')
```

Comment:
We need to remove `CNV_PRERELEASE_CATALOG_IMAGE` from the ci-operator/step-registry/hypershift/kubevirt/install/hypershift-kubevirt-install-ref.yaml file if we expect to detect that env var a different way

Reply:
yes, makes sense.
the idea is that `CNV_PRERELEASE_CATALOG_IMAGE` would be provided dynamically via jenkins+gangway automation.
if it's not provided and `CNV_PRERELEASE_VERSION` is provided, the nightly channel will be used.
if neither are provided, the production CNV version will be used.